misc/alert_rules.json

```diff
@@ -85,12 +85,12 @@
     "severity": "critical"
   },
   {
-    "rule": "macros.state_sensor_critical && sensors.sensor_alert = 1",
```

Comment:
Did this cause some kind of issues? This should be working fine if the sensor has the alert-flag set. This option allows ignoring sensors per device.

Reply:
Hello
If you remove this, then you have no more option to disable alerting on this specific sensor, and you break the alerting logic. So could you please revert this change ?
Thanx